frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -300,7 +303,9 @@ const commonTemplatesUpdater = ({ id, prevState, dispatch, getState }: UpdateOpt
   const [data, isBase64] = CloudInitDataHelper.getUserData(
```

Comment:
hasVMSettingsValueChanged can take multiple arguments

Reply:
it is not iCloudInitStorage, but cloudInitStorage. Please use `iGetStorages` to get `iCloudInitStorage`.